blockchain/types/tx_internal_data.go

```diff
@@ -672,3 +677,62 @@ func calculateTxSize(data TxInternalData) common.StorageSize {
 	rlp.Encode(&c, data)
 	return common.StorageSize(c)
 }
+
+func ValidateAccountState(stateDB StateDB, xType TxType, addr common.Address) bool {
```

Comment:
Please make it more specific because technically there are other "account state" checks like CheckReplacable.
```suggestion
func validate7702(stateDB StateDB, txType TxType, from, to common.Address) bool {
```

Reply:
According to KIP, it seems that checking the type of tx requires checking either "to" or "from" depending on the tx, but is it necessary to check "to" and "from"?

Reply:
No I meant to make this function somewhat generic, so that callers can call `validate7702(...., t.From, t.To) // ValueTransfer and SmartContractExecution` or `validate7702(..., t.From, common.Address{}) // AccountUpdate`.

In this function, group 1 and 3 checks for `to` and group 2 checks for `from`.